Bottom: fb3693eefef5e9d75f323cb7d6edab9641719282
Top:    37a843b045d17d5316961aa1c771df27c7cf3ccf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 10:45:38 -0500

Refresh of ldap-haproxy

---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 38e7fdc6b..11afcddab 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -51,3 +51,5 @@ haproxy_port_args:
 - "{{ buildbot_ip_addr }}:443:443"
 - "{{ nexus_ip_addr }}:80:80"
 - "{{ nexus_ip_addr }}:443:443"
+- "{{ phpldapadmin_ip_addr }}:80:80"
+- "{{ phpldapadmin_ip_addr }}:443:443"
